### ISSUE TITLE
Unable to type in the command palette input box

### ISSUE THREAD

**Issue by karthiknadig** [microsoft/vscode#186441]
Testing #186217

https://github.com/microsoft/vscode/assets/3840081/34e41c74-5cd3-42c1-aa37-dc7cf2c4df2d

This is on Windows, works fine on linux.
I pressed ctrl+shift+p, and started typing "o". That is when the narrator says "not available", because the focus is on the first quick pick item, and not on the input box. After every character I need to hit shift+tab to go back to input box to type.
